src/sonic_ax_impl/mibs/vendor/cisco/bgp4.py

```diff
@@ -38,7 +36,7 @@ def update_data(self):
             neigh_str = neigh_key
             neigh_str = neigh_str.split('|')[1]
             neigh_info = self.db_conn[db_index].get_all(mibs.STATE_DB, neigh_key, blocking=False)
-            if neigh_info is not None:
+            if neigh_info is not None and 'state' in neigh_info:
```

Comment:
>state [](start = 43, length = 5)

@SuvarnaMeenakshi if 'state' is missing, is it a bug in upstream process?
If it is valid use case, what does it mean?

Reply:
We would like to see if this will fix your test.
```suggestion
            if neigh_info:
```

Reply:
Thanks. I'll update and verify.

Reply:
Updated. Thanks